en/docs/key-concepts/identity-provisioning-and-its-standards.md

```diff
@@ -107,19 +100,17 @@ specific emphasis on simplicity and interoperability.
 	   played a major role in building the SPML 1.0. The following are
 	   the three proprietary provisioning standards.
 
-	   -   [Information Technology Markup Language
-    	   (ITML)](http://xml.coverpages.org/itml.html)
-	   -   [Active Digital Profile (ADPr)](http://xml.coverpages.org/adpr.html)
-	   -   [eXtensible Resource Provisioning Management
-    	   (XRPM)](http://xml.coverpages.org/xrpm.html)
+       -   <a href="http://xml.coverpages.org/itml.html" target="_blank">Information Technology Markup Language(ITML)</a>
+	   -   <a href="http://xml.coverpages.org/adpr.html" target="_blank">Active Digital Profile (ADPr)</a>
+	   -   <a href="http://xml.coverpages.org/xrpm.html" target="_blank">eXtensible Resource Provisioning Management
+    	   (XRPM))</a>
 
-	   [SPML
-	   1.0](https://www.oasis-open.org/committees/download.php/3032/cs-pstc-spml-	core-1.0.pdf)
-	   defined a Request/Response protocol as well as couple of bindings.
+	   <a href="https://www.oasis-open.org/committees/download.php/3032/cs-pstc-spml-	core-1.0.pdf" target="_blank">SPML
+	   1.0</a> defined a Request/Response protocol as well as couple of bindings.
 	   Requests/Responses are all based on XML and each operation has it own
 	   schema.
 
-	   One of the [bindings](http://xml.coverpages.org/SPML-Bindings10-OS.pdf)
+	   One of the <a href="http://xml.coverpages.org/SPML-Bindings10-OS.pdf" target="_blank">bindings</a>
```

Comment:
We don't need to add target="_blank". We need to add rel="noopener" instead to external links. Please fix in all the relevant places.